### DISCUSSION TITLE
VueDevtools is expecting to see <script src="http://localhost:11111"></script> at top of page

### DISCUSSION THREAD

**Discussion by chrissyast** [quasarframework/quasar#16939] [General - Components / Directives / etc]
![image](https://github.com/quasarframework/quasar/assets/31564153/6d9b9877-1745-441c-a758-905c55028824)

But where in my file structure am I supposed to put it? I have tried in App.vue and index.html but it still doesn't work

**Comment by rstoenescu** [comment]
You haven't mentioned what CLI you are using and which version. And what Quasar mode are you developing.
But regardless, I will answer with a tip assuming that you've configured the devServer to `open: true` (otherwise said: you're opening the devserver URL with a browser). If the vue devtools still says it is waiting for a connection, try to refresh the browser tab.
